### ISSUE TITLE
Access CDN and make corrections to header footer on support page

### ISSUE THREAD

**Issue by ReneeRendon** [rackerlabs/rackspace-how-to#1523]
The How-To team needs access to CDN, so we can update the header and footer sections of the support.rackspace.com landing page.  Once we have access, we need to correct the following items:
- [x] Feedback link at the very top does not take you to a place to leave feedback.
- [x] Hovering over the green Sign Up button (top right) allows you to sign up for Cloud Sites. Cloud Sites needs to be removed.
- [x] Hovering over the gray Log In button, allows you to access Cloud Sites CP. Cloud Sites needs to be removed.
- [x] In the footer, API documentation takes you to docs.rackspace.com
- [x] In the footer, The Rackspace Blog and the Developer Blog are listed twice in the footer.
- [x] In the footer, Case Studies and Customers Stories both take you to Customer Stories.
- [x] In the footer, New Products = 404.
- [x] In the footer, Feedback redirects from UserVoice to main Rackspace page.
- [x] Remove KC banner

**Comment by catlook** [comment]
Here's perhaps source for banner...not sure where it's actually pulled in and displayed.  The hunt goes on.
./templates/developer.rackspace.com/_includes/header-nav.html:56:                            <li><a href="">Knowledge Center</a></li>
./templates/developer.rackspace.com/_includes/header-nav.html:105:                            <li><a href="">Knowledge Center</a></li>

**Comment by catlook** [comment]
Feedback in Nexus:

Correct: /templates/support.rackspace.com/_includes/feedback-sidebar.html  (points to https://community.rackspace.com/feedback/f/68)

Bad:
/templates/developer.rackspace.com/images.html:117: (points to https://feedback.rackspace.com/)

Contains some cloud sites stuff:
/templates/developer.rackspace.com/_includes/site-header.html
/templates/developer.rackspace.com/common/header.hbs:45:

**Comment by catlook** [comment]
I surrender..super low priority so assigning myself.

**Comment by nimbinatus** [comment]
I'll claim this one as I'm working on the KC banner anyway

**Comment by nimbinatus** [comment]
For all but the KC banner, the changes have been posted as a PR on the enterprise repo

**Comment by nimbinatus** [comment]
made a PR on the original repo to fix the issue that was breaking the build and made a PR that will fix the banner. Ref PR #1425

**Comment by nimbinatus** [comment]
Still waiting on approval to the last PR, and then for one of the owners to push the whole patch (all 9 issues) up to the CDN.

**Comment by nimbinatus** [comment]
Got all the approvals and the PR has been merged. Now we're tracking down the CDN credentials. Found some contacts with Margaret, and we're en route.

**Comment by nimbinatus** [comment]
I've got access now. Waiting to push the fixes until after business hours (Central Time Zone)

**Comment by nimbinatus** [comment]
Changes posted. Please clear your cache if you still see any of these problems.
